File: Inventarization.Api/Inventorisation/Models/Action.cs
using Inventorization.Business.Model;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Inventorization.Api.Models
{
    public class Action
    {
        public Guid Id { get; set; }
        public DateTime DateTime { get; set; }
        public ActionType Type { get; set; }
        public Guid Inventorization { get; set; }
        public string User { get; set; }
        public string BarCode { get; set; }
        public ZoneViewModel Zone { get; set; }
        public int Quantity { get; set; }
        public bool FoundInItems { get; set; }
        public string Description { get; set; }
        public string Name { get; set; }
        
    }

    public class SaveActionVM
    {
        public Guid? Id { get; set; }
        public DateTime? DateTime { get; set; }
        public ActionType Type { get; set; }
        public string BarCode { get; set; }
        public int Quantity { get; set; }
        public string Description { get; set; }
        public Guid Zone { get; set; }
    }

    public class DeleteModel
    {
        public Guid Id { get; set; }
    }

}

File: Inventarization.Android.Client/app/src/main/java/inventory/aeco/network/models/Action.java
package inventory.aeco.network.models;

import java.util.Date;

import inventory.aeco.network.models.ActionStatus;
import inventory.aeco.ActionType;

/**
 * Created by Барашики on 04.04.2017.
 */


public class Action {
    public String Id;
    public Date DateTime;
    public ActionType Type;
    public String User;
    public String BarCode;
    public String Zone;
    public String Inventorization;
    public Integer Quantity;
    public ActionStatus Status;
}
